guacamole/src/main/webapp/app/rest/types/Error.js

```diff
@@ -43,6 +43,15 @@ angular.module('rest').factory('Error', [function defineError() {
         this.message = template.message;
```

Comment:
Where is this message field actually used? It looks like it's been replaced universally by translatableMessage.